src/clusterfuzz/_internal/protos/untrusted_runner.proto

```diff
@@ -229,9 +229,7 @@ message StatRequest {
 
 message CrossPollinationStats {
   optional string project_qualified_name = 1;
-  optional string method = 2;
```

Comment:
@oliverchang did i do this correctly?
I reran the generate.sh script after this edit by the way.

Reply:
LGTM.